Interface/AddOns/oUF_Neav/core.lua

```diff
@@ -421,7 +421,13 @@ local function UpdateHealth(Health, unit, cur, max)
     if (UnitIsDeadOrGhost(unit) or not UnitIsConnected(unit)) then
         Health:SetStatusBarColor(0.5, 0.5, 0.5)
     else
-        Health:SetStatusBarColor(0, 1, 0)
+        if (UnitIsPlayer(unit) and config.show.classHealth) then
+            local localizedClass, englishClass = UnitClass(unit)
```

Comment:
Since you're not using `localizedClass`, assign it to `_`.

```diff
-local localizedClass, englishClass = UnitClass(unit)
+local _, englishClass = UnitClass(unit)
```